fixieai/client/client.py

```diff
@@ -382,3 +377,17 @@ def deploy_agent(
             gzip_tarfile=gzip_tarfile,
             environment=environment,
         )
+
+    def post(self, url: str, data: Optional[Dict[str, Any]] = None) -> Any:
```

Comment:
Since these APIs are part of our SDK documentation, do you mind adding docstrings here? Also for streaming_post.